bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -258,15 +263,21 @@ public void configureInstallationInstance(BigDecimal installationId)
         }
     }
 
-    private void setPasswordFromCookie() {
+    private void analyzeCookies() {
         CookieStore c = httpClient.getCookieStore();
         List<HttpCookie> cookies = c.getCookies();
         final List<HttpCookie> unmodifiableList = List.of(cookies.toArray(new HttpCookie[] {}));
         unmodifiableList.forEach(cookie -> {
             logger.trace("Response Cookie: {}", cookie);
-            if (cookie.getName().equals(PASSWORD_NAME)) {
-                password = cookie.getValue();
-                logger.debug("Fetching vid {} from cookie", password);
+            if (cookie.getName().equals(VID)) {
```

Comment:
You certainly have a SAT warning telling you to reverse and put the constant at left:  VID.equals(cookie.getName())
Same for the two others below

Reply:
Fixed!